app/views/partials/alerts/list_alarm_callbacks.scala.html

```diff
@@ -27,7 +27,11 @@
 
             <h3>
                 <i class="icon icon-ellipsis-vertical"></i>
-                @availableAlarmCallbacks.get(callback.getType).name
+                @if(availableAlarmCallbacks.get(callback.getType) != null) {
```

Comment:
Since I don't know the complexity of retrieving the alarm callbacks from the backend, I'd suggest putting `availableAlarmCallbacks.get(callback.getType)` into a variable as it's used again in the next conditional block.